vizro-core/src/vizro/static/css/table.css

```diff
@@ -1,93 +1,38 @@
-#right-side .dash-cell {
-  height: 48px !important;
+#dashboard-container .dash-cell {
```

Comment:
Why is every selector in this file prefixed with `#dashboard-container`, and we don't do the same in other static/css files? Is the long-term goal to standardise by prefixing all our static/css selectors with #dashboard-container?

Reply:
We do it in other files as well such as the dropdown and aggrid. However, not every file needs this. It's only needed when we have specificity issues with the `dash` components. So you need to have this prefix, such that our CSS is more specific and therefore has precedence :)